src/api/app/controllers/webui/package_controller.rb

```diff
@@ -831,26 +831,24 @@ def check_package_name_for_new
   end
 
   def find_last_req
-    if @oproject && @opackage
-      last_req = BsRequestAction.where(target_project: @oproject,
-                                       target_package: @opackage,
-                                       source_project: @package.project,
-                                       source_package: @package.name).order(:bs_request_id).last
-      return unless last_req
-
-      last_req = last_req.bs_request
-      if last_req.state != :declined
-        return # ignore all !declined
-      end
+    return if @oproject.blank? && @opackage.blank?
 
-      return {
-        id: last_req.number,
-        decliner: last_req.commenter,
-        when: last_req.updated_at,
-        comment: last_req.comment
-      }
-    end
-    nil
+    last_req = find_last_bs_request_not_declined
+
+    return if last_req.blank?
+
+    { id: last_req.number, decliner: last_req.commenter,
```

Comment:
aehm, do you realize why it says decliner here? Hint: this is not about not_declined

Reply:
ähhm was meinst du @coolo 👼